_gsocblogs/2022/blog_PODIO_SoumilBaldota.md

```diff
@@ -3,44 +3,73 @@
 project: HSF
 title: Interfacing PODIO with Julia
 author: Soumil Baldota
-date: 28.07.2022 
+date: 4.09.2022 
 year: 2022 
 layout: blog_post
-logo: hsf_logo_angled.png
-intro: Midterm progress report
+logo: podio-logo.png
+intro: Final progress report

```

Comment:
The intro is what is visible on the front page with many posts, it is an incentive for visitors to read more, so it should not look like a title.

Reply:
If you don't want to use a photo, we encourage using an avatar image of your choice, you can even use a web-based one such as your GitHub avatar that you can update later, see this [example](https://hepsoftwarefoundation.org/gsoc/blogs/2022/blog_HSF_TheAdmins.html)

Reply:
I have added recent commit addressing this.